runtime/cudaq/platform/default/rest_server/helpers/JsonConvert.h

```diff
@@ -9,8 +9,8 @@
 #pragma once
 #include "common/ExecutionContext.h"
 #include "common/FmtCore.h"
+#include "cudaq/Support/Version.h"
 #include "nlohmann/json.hpp"
-
 /*! \file JsonConvert.h
```

Comment:
```suggestion
/*! \file
```
It ought to work to omit the filename. Easier to maintain.